common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -150,6 +150,15 @@ class VideoFields(object):
         display_name=_("Upload Handout"),
         scope=Scope.settings,
     )
+    only_on_web = Boolean(
+        help=_(
+            "Specify whether access to this video is limited to browsers only, or if it can be "
+            "accessed from other applications including mobile apps."
```

Comment:
I've always seen these types of help strings on one line. Is it okay to split this?